.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,6 +31,20 @@ For example:
 
 - [ ] Verify that no errors appear in the JS console
 
+### C+ PR Review Checklist ([full steps here](../PR_REVIEW_CHECKLIST.md))
+<!--
+This is a checklist for Contributor Plus (C+) reviewers. If the PR is internal, these do not need to be checked off, but are still a nice reminder for what to look for when reviewing PRs.
+-->
+- [ ] Are screenshots or videos included for tests on all platforms? ([required](https://github.com/Expensify/App/blob/main/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms))
+- [ ] I (the C+ assigned to review this PR) also tested this PR on **all platforms**
+- [ ] Is the correct issue linked in the `### Fixed Issues` section?
+- [ ] Are testing steps clear and do they cover the changes + possible regressions?
+- [ ] Do testing steps cover success & fail scenarios (if applicable)?
+- [ ] There are no console errors related to changes in this PR
+- [ ] Appropriate new regression steps are mentioned below (if applicable)
+- [ ] Code patterns were checked (see `Reviewing the code`)
```

Comment:
For this line and the next, can we link directly to the docs section instead of just using code blocks?

Reply:
Yeah I tried and failed last week, but turns out it was very easy :D